### PR TITLE
DDF-1225 Move sdk project under distribution

### PR DESCRIPTION
**Pull request by jckilmer**
@coyotesqrl @rzwiefel @roelens8 @pklinef @tbatieConnexta

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/81)

### CODE REVIEW

**Comment by coyotesqrl** [comment]
Fails build.

File: distribution/pom.xml [hunk 1]
```diff
@@ -50,6 +50,7 @@
         <module>ddf</module>
         <module>test</module>
         <module>docs</module>
+        <module>sdk</module>
```
Comment (pklinef) [inline distribution/pom.xml]: SDK needs to be built before test since the itests use them.

**Comment by coyotesqrl** [comment]
LGTM!

:+1:

File: distribution/sdk/pom.xml [hunk 2]
```diff
@@ -17,11 +17,11 @@
...
-        <artifactId>ddf-parent</artifactId>
-        <version>3.0.4-SNAPSHOT</version>
+        <artifactId>distribution</artifactId>
+        <version>2.8.0-SNAPSHOT</version>
     </parent>
 
-    <groupId>ddf.sdk</groupId>
+    <groupId>ddf.distribution</groupId>
     <artifactId>sdk</artifactId>
     <version>2.8.0-SNAPSHOT</version>
```
Comment (shaundmorris) [inline distribution/sdk/pom.xml]: Do you need this 2.8.0-SNAPSHOT version? I would have expected this pom to now inherit this version from distribution/pom.xml.

**Comment by roelens8** [comment]
Looks good.

---







Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/81)

**Comment by roelens8** [comment]
Reviewed 87 of 89 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/81)

File: distribution/sdk/pom.xml (hunk 2 above)
Comment (pklinef) [inline distribution/sdk/pom.xml]: I agree, this version should be removed now.

File: pom.xml [hunk 3]
```diff
@@ -28,7 +28,6 @@
          If access to the previous tags are needed, please refer back to an older DDF repository
          before the transition over to a modular structure. -->
 	<modules>
-        <module>sdk</module>
         <module>distribution</module>
 	</modules>
```
Comment (pklinef) [inline pom.xml]: The build plugin should move into the distribution module.

**Comment by coyotesqrl** [comment]
Re-approved. :+1:

**Comment by roelens8** [comment]
:+1:

---







Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/81)

**Comment by roelens8** [comment]
Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 3 unresolved discussions, some commit checks broke.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/81)

**Comment by shaundmorris** [comment]
Verified build, did not see the same issues reported on bamboo currently. That still needs to be investigated.

**Comment by shaundmorris** [comment]
@jckilmer can you double check the artifactId/groupIds in the sdk-app features.xml file? I believe they should be changed from ddf.sdk to ddf.distribution.

**Comment by jckilmer** [comment]
@shaundmorris good catch, you're right, I'll create a ticket to fix the features.xml file.

**Pull request merged**